File: PR1/PR1_Ej1/src/BicicletaCarretera.java

public class BicicletaCarretera extends Bicicleta{

    public BicicletaCarretera(){
        tipo = "carretera";
    }
    @Override
    public void mostrarTipo() {
        System.out.println("Es una bici de " + this.tipo + " con id : " + this.id + "\n");
    }
}

File: PR1/PR1_Ej2/BicicletaCarretera.py
from Bicicleta import Bicicleta
import copy

class BicicletaCarretera(Bicicleta):
    def __init__(self):
        self.tipo="carretera"

    def mostrarTipo(self):
        print("Soy una bici de ", self.tipo,"\n")

    def clone(self):
        return copy.copy(self)


